test/unit/region.spec.js

```diff
@@ -1042,6 +1050,11 @@ describe('region', function() {
     it('should return the region', function() {
       expect(this.itemView.getRegion('MyRegion')).to.equal(this.region);
     });
+
+    it('should call render if getRegion is called without being rendered', function() {
+      this.itemView.getRegion('whoCares');
+      expect(this.itemView.render.called).to.be.true;
```

Comment:
`expect(this.itemView.render).to.be.calledOnce`  That's sinon-chai and will produce a better error than "expected `false` to be `true`"